src/server.h

```diff
@@ -456,6 +456,9 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 	static u16 getProtocolVersionMin();
 	static u16 getProtocolVersionMax();
 
+	// NOTE(obelisk) Exposed so `remoteinputhandler.cpp` can synchronize env access.
```

Comment:
```suggestion
	// Exposed so `remoteinputhandler.cpp` can synchronize env access.
```

Reply:
This seems useless if the other code that access the env object doesn't grab the mutex. Am I missing something?

Reply:
I left a note there because I found it helpful to know why code was changed in our fork when doing an upstream merge. Git blame isn't always as clear.

Reply:
> This seems useless if the other code that access the env object doesn't grab the mutex. Am I missing something?

which other code doesn't grab the mutex?

Reply:
Just mentioning remoteinputhandler is enough to make it clear I think. That file doesn't exist upstream.